pkg/mountkexec/mountkexec.go

```diff
@@ -39,6 +39,19 @@ func MountISO(isoPath, mountDir string) error {
 	return nil
 }
 
+// MountISOPmem creates a directory and mounts the pmem block device on the directory
+func MountISOPmem(Pmem, mountDir string) error {
+	if err := os.MkdirAll(mountDir, 777); err != nil {
+		return fmt.Errorf("error making mount directory:%v", err)
+	}
+	var flags uintptr
+	flags |= unix.MS_RDONLY
+	if err := mount.Mount(Pmem, mountDir, "iso9660", "", flags); err != nil {
```

Comment:
you can just replace the flags parameter in Mount with unix.MS_RDONLY and remove lines 47 and 48

Reply:
Done